src/main/java/net/mcreator/element/types/Item.java

```diff
@@ -98,6 +98,7 @@
 	// Ranged properties
 	public boolean enableRanged;
 	public boolean shootConstantly;
+	public boolean chargesPower;

```

Comment:
It is too late for other params, but this one could be `rangedItemChargesPower` or `rangedChargesPower` to distinguish from other item parameters